blocks/commerce-return-header/commerce-return-header.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-extraneous-dependencies */
+import { events } from '@dropins/tools/event-bus.js';
+import { Header, provider as uiProvider } from '@dropins/tools/components.js';
+import { CUSTOMER_RETURN_DETAILS_PATH, CUSTOMER_RETURNS_PATH } from '../../scripts/constants.js';
+
+// TODO - Will be refactored during implementation of return-details page
```

Comment:
I'd reference a ticket number in the TODOs that are going to be merged to ensure this is not forgotten later.

Reply:
Agree 👍 
Added ticket links.